README.md

```diff
@@ -1,184 +1,6 @@
-<a name="readme-top"></a>
```

Comment:
Please make sure the README of your project has more descriptive details **as this is the first thing anyone visiting you're repo will look at and having a detailed README will help them quickly set up and see your project on their machine**

Reply:
Good use of GitHub code review comment